src/components/homepage/HomepageSlider/HomepageSlider.module.scss

```diff
@@ -3,16 +3,54 @@
   left: 50%;
   transform: translate(0);
   width: 33vw;
-  height: 24.8rem;
+  height: 248px;
+
+  .control {
+    position: absolute;
+    top: 35%;
+    width: 50px;
+    height: 50px;
+    z-index: 2;
+  }
+
+  .prev,
+  .next {
+    background-size: 22px;
+    background-position: center;
+    background-repeat: no-repeat;
+    cursor: pointer;
+  }
+
+  .prev {
+    background-image: url(/img/arrow_left.svg);
+    left: -10rem;
+  }
+
+  .next {
+    background-image: url(/img/arrow_right.svg);
+    right: -10rem;
+  }
+
+  .prev:active,
+  .next:active {
+    transform: scale(0.8);
+  }
+
   .sliderWrapper {
     width: 33vw;
+    height: 248px;
```

Comment:
@hubert-deriv I would suggest replacing `px` with `rem` for the newly added styles where applicable and depending on how much pixels there is in 1 rem in this project 🙂